ai-ml/gke-ray/rayserve/llm/serve_tpu.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: this file was inspired from: https://github.com/richardsliu/vllm/blob/rayserve/examples/rayserve_tpu.py
+
+import os
+
+import json
+import logging
+from typing import Dict, List, Optional
+
+import ray
+from fastapi import FastAPI
+from ray import serve
+from starlette.requests import Request
+from starlette.responses import Response
+
+from vllm import LLM, SamplingParams
+
+logger = logging.getLogger("ray.serve")
+
+model_id = "meta-llama/Meta-Llama-3-70B"
```

Comment:
Can we fetch the model ID using an envrionment variable? This way you can configure the model ID using runtime env

Reply:
Oh I see the check in build_app for MODEL_ID now, it's probably fine to just make the environment required and remove this variable

Reply:
Done in [335e75c](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/1467/commits/335e75c1656947c45eee2b26a99605e8e12211b8).